eq-author/src/App/page/Preview/QuestionPagePreview.test.js

```diff
@@ -20,6 +21,27 @@ describe("QuestionPagePreview", () => {
 
   actSilenceWarning();
 
+  const commentsSubscription = gql`
```

Comment:
You could import the query from this file `eq-author/src/components/CommentsPanel/commentSubscription.graphql`